Detections/MultipleDataSources/ExtensiveAWSComputeResurseDeployements.yaml

```diff
@@ -0,0 +1,54 @@
+id: 9e457dc4-81f0-4d25-bc37-a5fa4a17946a

```

Comment:
This is in the MultipleDataSources folder but I only see it using CloudTrail, wouldn't it be better in the AWS solution?